config/environments/development.rb

```diff
@@ -21,7 +21,7 @@
       'Cache-Control' => 'public, max-age=172800'
     }
   else
-    config.action_controller.perform_caching = false
+    config.action_controller.perform_caching = true
```

Comment:
http://blog.bigbinary.com/2016/01/25/caching-in-development-environment-in-rails5.html